execution/execution.go

```diff
@@ -38,12 +38,12 @@ func (s Status) String() (r string) {
 // Execution stores all informations about executions.
 type Execution struct {
 	ID                string                 `hash:"-"`
-	EventID           string                 `hash:"eventID"`
+	EventID           string                 `hash:"name:eventID"`
```

Comment:
@krhubert can you add a simple test that create 2 execution with different data and compare their hash?